README.md

```diff
@@ -1,41 +1,50 @@
-- Team [Engineering Practices](.github/EngineeringPractices.md)
-- CSE Engineering Fundamentals [Playbook](https://github.com/Microsoft/code-with-engineering-playbook)
-
-## How to file issues and get help  
-
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new issue.
-
-For help and questions about using this project, please open a GitHub issue.
-
-## Contributing
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit <https://cla.opensource.microsoft.com>
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Trademarks
-
-This project may contain trademarks or logos for projects, products, or services.
-
-Authorized use of Microsoft trademarks or logos is subject to and must follow [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
-
-Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
-
-Any use of third-party trademarks or logos are subject to those third-party's policies.
+# RelayRunner App
+
+RelayRunner App is intended to faciliate testing in controlled environments by adding the capability to update configs and load tests without restarting load clients.
+
+## Prerequisites
+
+- Bash shell (tested on Visual Studio Codespaces, Mac, Ubuntu, Windows with WSL2)
+  - Will not work with WSL1 or Cloud Shell
+- Azure CLI ([download](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- Docker CLI ([download](https://docs.docker.com/install/))
+- .NET 5.0 ([download](https://docs.microsoft.com/en-us/dotnet/core/install/))
+- Visual Studio Code (optional) ([download](https://code.visualstudio.com/download))
+
+## RelayRunner Usage
+
+TODO
+
+## Run the Application
+
+### Using Visual Studio Codespaces
+
+> Visual Studio Codespaces is the easiest way to evaluate RelayRunner.
+
+TODO: Describe opening and running a codespaces
+
+### Using bash shell
+
+TODO
+
+### Testing the application
+
+TODO
+

```

Comment:
@abassey2, please add a little into the readme on how to build, run, and test.  I know it is just the start of the scaffolding, but if we have others start helping we want to leave them with some getting started directions :)